blog_system_db_data.json

```diff
@@ -1,350 +1,350 @@
+        "pk": 6,
+        "fields": {
+            "owner": 4,
+            "title": "What is iteration",
+            "content": "Iteration, in the context of computer programming, is a process wherein a set of instructions or structures are repeated in a sequence a specified number of times or until a condition is met. When the first set of instructions is executed again, it is called an iteration",
+            "created_time": "2022-11-23T14:19:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 1,
+        "fields": {
+            "user": 2,
+            "post": 1,
+            "content": "Cool!",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 2,
+        "fields": {
+            "user": 3,
+            "post": 1,
+            "content": "Wooow",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 3,
+        "fields": {
+            "user": 4,
+            "post": 1,
+            "content": "Interesting",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 4,
+        "fields": {
+            "user": 6,
+            "post": 1,
+            "content": "Not bad",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 5,
+        "fields": {
+            "user": 6,
+            "post": 3,
+            "content": "I didn't know that",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 6,
+        "fields": {
+            "user": 3,
+            "post": 4,
+            "content": "I am interested:)",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 7,
+        "fields": {
+            "user": 1,
+            "post": 4,
+            "content": "I love Python more. Python is a better-designed language that makes it easy to maintain, whereas JavaScript is poor. Python is not good for mobile development, whereas Java-Script is good. Python is slow to run compared to JavaScript. Python provides a huge standard library, whereas JavaScript has a limited standard library.",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 8,
+        "fields": {
+            "user": 1,
+            "post": 5,
+            "content": "I knew that!",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
+    },
+    {
+        "model": "blog.Commentary",
+        "pk": 8,
+        "fields": {
+            "user": 5,
+            "post": 6,
+            "content": "I had that question at tech-check in IT-company...",
+            "created_time": "2022-11-23T16:28:09Z"
+        }
```

Comment:
There is a duplicate primary key (pk) for the `Commentary` model with pk 8. This could lead to issues when loading the data into the database. Consider changing the pk to a unique value.